### ISSUE TITLE
Fix required fields in contact us forms to be required

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#167]
Address and username in the long form did not have the appropriate attributes to be required. The short form is also missing the aria-required attribute. Reported by @danthibodeau
